apf/apfH1Shapes.cc

```diff
@@ -0,0 +1,769 @@
+/*
+ * Copyright 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+#include "apfShape.h"
+#include "apfMesh.h"
+#include "apfFieldOf.h"
+#include "apfElement.h"
+#include "apfVectorElement.h"
+#include "apfPolyBasis1D.h"
+#include <mth.h>
+#include <mth_def.h>
+#include <mthQR.h>
+#include <pcu_util.h>
+#include <PCU.h>
+
+#include <iostream>
+using namespace std;
```

Comment:
Is this needed?  I don't see much stl usage below.

Reply:
Fixed this.